src/Compilers/CSharp/Test/Emit2/Semantics/PatternMatchingTests.cs

```diff
@@ -3362,8 +3361,7 @@ public static void Main(string[] args)
             compilation.VerifyDiagnostics(
                 // (7,13): error CS8059: Feature 'pattern matching' is not available in C# 6. Please use language version 7.0 or greater.
                 //             case 1 when true:
-                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion6, "case 1 when true:").WithArguments("pattern matching", "7.0").WithLocation(7, 13)
-                );
+                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion6, "case").WithArguments("pattern matching", "7.0").WithLocation(7, 13));
```

Comment:
What was the rationale for changing the location of some pattern diagnostics?

Reply:
large diagnostics are highly problematic, squiggling lots of the screen, blocking other narrower diagnostics, and leading to difficult to interact with lightbulbs and other issues.  Diagnostics should be targeted as much as possible, isolating them to a token when sensible to do so.